Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -1152,11 +1152,6 @@ public void PauseDirtyTracking()
 
         public void ResumeDirtyTracking()
         {
-            if (DirtyTracking == eDirtyTracking.NotStarted)
-            {
-                StartDirtyTracking();
-                return;
-            }
```

Comment:
remove this as if we haven't start tracking an object, I don't want to track it now. this would cause unit tests to fail due to exception was thrown because of that part
checked all references for that method and all of them are by me from my recent changes, probably was an obsolete method from previous Ginger versions